documentation/source/building.rst

```diff
@@ -301,6 +301,15 @@ Makefile-based Test Scripts
     You can usually write out such a file from the simulator's GUI.
     This is currently supported for the Mentor Questa, Mentor ModelSim and Aldec Riviera simulators.
 
+.. make:var:: TOPLEVEL_LANG
+
+    Used to inform the makefile scripts which HDL language the top-level design element is written in.
+    Currently it supports the values ``verilog`` for Verilog or SystemVerilog tops, and ``vhdl`` for VHDL tops.
+    This is used by simulators that support more than one interface (VPI, VHPI, or FLI) to select the appropriate interface to start cocotb.
+
+    The variable is also made available to cocotb tests conveniently as :data:`cocotb.LANGUAGE`.
+    This can be used to prevent individual tests from running with the selected :make:var:`TOPLEVEL_LANG`.
+
```

Comment:
Should this be right next to `TOP_LEVEL`? This isn't optional, like all the other variables in this section.

Reply:
Or perhaps vice versa...

Reply:
Moved to right before `VERILOG_SOURCES`.